calcrule_social_protection/converters/__init__.py

```diff
@@ -0,0 +1,8 @@
+from calcrule_social_protection.converters.beneficiary import (
+    BeneficiaryToBillConverter,
+    BeneficiaryToBillItemConverter
+)
+from calcrule_social_protection.converters.group_beneficiary import (
+    GroupToBillConverter,
+    GroupToBillItemConverter
+)
```

Comment:
There are a lot of similarities between XToBillConverter and XToBillItemConverter. Can you use something like builder pattern to have one structure for each Bill and BillItem builders and then make adjustments in the implementation for the Beneficiaries and Groups? It'll be easier to maintain in future as well as extend by other resource types.

Reply:
changed.